README.md

```diff
@@ -26,7 +26,8 @@ To run the CSAF CMS server you need the following:
 You can find an example setup for local development in the 'compose.yaml' and 
 an example configuration for Keycloak in 'keycloak/csaf-realm.json'. You 
 can take this as a starting point, but please check the documentation of the
-individual projects for a proper production setup.
+individual projects for a proper production setup. We also recommend 
+running behind some kind of reverse proxy.
```

Comment:
It is unclear here, what should be run behind the reverse proxy. Maybe, also referring to the diagram could help? (If that describes the setup correctly: https://github.com/secvisogram/csaf-cms-backend/blob/main/documents/BSISecvisogramArchitecture.drawio.svg)